docs-src/ref-programs.scrbl

```diff
@@ -1996,3 +1996,39 @@ There are convenience methods defined for comparing fixed point numbers:
 @index{fxeq} @reachin{fxeq(x, y)} tests whether @tt{x} is equal to @tt{y}.
 
 @index{fxne} @reachin{fxne(x, y)} tests whether @tt{x} is not equal to @tt{y}.
+
+@subsubsection{@tt{Signed Integers}}
+
+The standard library provides abstractions for dealing with signed integers. The following definitions
+are used to represent @reachin{Int}s:
+
+@(mint-define! '("Int") '("Positive") '("Negative"))
+@reach{
+  const Int = { sign: bool, i: UInt };
+  const Positive = true;
+  const Negative = false;  }
```

Comment:
`Pos` and `Neg` abbreviations?

Reply:
Maybe hack the compiler so that unary plus/minus are the constants... maybe that's a bad idea?

Reply:
That would be nice, I'll look into that

Reply:
I made it so `FixedPoint` are signed and are a similar shape to `Int`, `{ sign: bool, i: a }`. Now, unary operations will work on both types. Additionally, the unary ops will cast values of type `UInt` to `Int` before applying the operation. The values can now be desugared from:

```javascript
4     // : UInt
+4    // : Int
-4    // : Int
4.0   // : FixedPoint
-4.0  // : FixedPoint
+4.0  // : FixedPoint
```